valkey.conf

```diff
@@ -386,6 +386,12 @@ databases 16
 # ASCII art logo in startup logs by setting the following option to yes.
 always-show-logo no
 
+# To prevent sensitive user information, such as PII, from being recorded in the 
+# server log file, this feature is enabled by default. If you need to log user 
+# data for debugging or troubleshooting purposes, you can disable this feature
+# by changing the config value to -1.
+hide-user-data-from-log 1
```

Comment:
Can we just set hide-user-data-from-log as 0 to disable this feature? In valkey.conf, the argument min-replicas-to-write, client-output-buffer-limit. min-replicas-to-write etc are set 0 as disable.  Then for other part, maybe the logic maybe easier to work